assets/css/admin-lesson.css

```diff
@@ -1,19 +0,0 @@
-/**
```

Comment:
This file is probably needed as the UI has an issue:
![Screenshot 2020-09-02 at 1 30 47 PM](https://user-images.githubusercontent.com/53191348/91970629-9ef7d580-ed20-11ea-88be-3411290d9cf9.png)

Reply:
@gkaragia It is still there, just moved to `admin-lesson.scss`. Git didn't track it as a rename for some reason. I think this is a bug in the style. When I inspect the interval number field, I can see the style applied from the generated `admin-lesson.scss`.

Reply:
A-ha! I see what happened. There was already an `admin-lesson.scss` file, however the css file was getting updates that the SCSS file wasn't. I'll bring the changes over to the scss file.